common/djangoapps/enrollment/views.py

```diff
@@ -81,34 +76,12 @@ def has_api_key_permissions(self, request):
 
 class EnrollmentUserThrottle(UserRateThrottle, ApiKeyPermissionMixIn):
     """Limit the number of requests users can make to the enrollment API."""
-    # TODO: After confirming that reducing the throttle is successful, remove
-    # and clean up waffles. The rate limit has been increased over the course
-    # of a few months to account for unnecessary calls from the ecommerce
-    # service. These calls are no longer made and the plan is to set the
-    # rate limit back to its original state. LEARNER-5148
-
     THROTTLE_RATES = {
         'user': '40/minute',
-        'staff': '2000/minute',
+        'staff': '120/minute',
```

Comment:
Sounds like there is some uncertainty around the staff level.  Can you add a comment in the code to document the uncertainty (minus the graphs) and point to the PR for more details?

Reply:
@robrap  I've added a comment that points to this PR. I think linking the PR should be sufficient. Thoughts?